clientServiceClient/api.go

```diff
@@ -141,3 +141,17 @@ type InternalRollQueenClientRequest struct {
 	RegistrationToken string             `json:"token"`
 	Client            ClientRegisterInfo `json:"client"`
 }
+
+type ClientListRequest struct {
+	NextToken int64 `json:"next_token"`
+	Limit     int   `json:"limit"`
+}
+
+type ClientListResponse struct {
+	ClientIDs []ClientID `json:"clients"`
+	NextToken int64      `json:"next_token"`
+}
+
+type ClientID struct {
```

Comment:
what's the point of this struct? If it wants to be future proof, it should be called something like ClientDetails, otherwise defining 151 to be `ClientIDs []string` is what I recommend